bs_detector/bias.py

```diff
@@ -19,6 +21,12 @@ def get_list_from_file(file_name):
     return l
 
 
+def get_text_from_article_file(article_file):
```

Comment:
I think this argument should be `article_file_path` since you call `open` on it. `article_file` to me sounds like I should have already opened the file.

Reply:
Agree